index.html

```diff
@@ -9,44 +9,45 @@
 
 </head>
 <body>
-    <section class="section-one">
         <header>
-    
-            <div class="header-containt">
-    
-                <nav>
-                    <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
-                    </ul>
-    
-                </nav>
-    
-            </div>
+          <div class="nav-list">
+            <nav>
+              <ul class="nav-menu">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+              <button class="nav-icon">
+                <i class="fas fa-bars"></i>
+              </button>
+            </nav>
+          </div>
         </header>
         <section class="hero-comtain">
```

Comment:
- [ Required ] I think it would be better if your design has `paddings`, `margins`, and `background-image` that match the Figma design provided for this milestone as requested by the previous reviewer. Currently, your design varies from the Figma template in the following ways.
   
  - [ ]  The top part of the background image appears to have been cut off and is not visible in your design, while on the Figma template, this top part is visible.
   - [ ] The white space between the `Hey there I'm Roland` on your design appears to be too big compared to the white space on the Figma design.

## Screenshots
| Figma template | Your design |
|----------------------|------------------|
|![251838031-57a3e5d0-f191-45fa-8623-31999bf360b3](https://github.com/RolandAmini/portfolio-mobile/assets/16392046/fb554279-2ea8-4868-891a-012479766e54)| ![Screenshot from 2023-07-08 14-06-00](https://github.com/RolandAmini/portfolio-mobile/assets/16392046/19c737bb-a486-47db-adf5-9be5ce7177f1)|

## Example code
I would like to suggest that you refactor your code as shown below
```CSS
/* Remove this stying that makes the header fixed */
header {
   position: fixed;
   left: 0;
   right: 0;
   top: 0; 
}

/* Change the styling for the .hero-comtain element to this */
.hero-comtain {
    display: flex;
    flex-direction: column;
    align-items: center;
    text-align: center;
    padding: 0px 50px 0 50px;
    background-image: url(header-illsutration-mobile.svg);
    background-repeat: no-repeat;
    background-size: cover;
    height: 70vh;
}
```